lib/core/test/unit/Cardano/Wallet/Api/Malformed.hs

```diff
@@ -1151,6 +1151,13 @@ instance Malformed (BodyParam ApiMaintenanceActionPostData) where
           )
         ]
 
+instance Malformed (BodyParam (ApiT SmashServer)) where
+    malformed = first (BodyParam . Aeson.encode) <$>
```

Comment:
Is it still relevant?

Reply:
I guess we don't have query param specs yet:

```
    -- TODO
    -- Capture request query params as QueryParam
    --
    -- newtype QueryParam t = QueryParam Text
    --     deriving (Typeable)
    --
    -- type MkQueryRequest api :: *
    -- gEveryQueryParams :: Proxy api -> Request -> MkQueryRequest api
```

Reply:
Indeed we do not, but the malformed instances are used only if the underlying type is used in the API. And now, there is no API call using `ApiT SmashServer` as a body parameter.